### PR TITLE
python: include example using pip

### PR DESCRIPTION
**Pull request by nuno-agostinho**
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by sbrl — COMMENTED** [review]
Thanks for the edit! However, isn't there a `pip`/`pip3` command too? Which one is the preferred mechanism for calling `pip`? I'm hesitant to accept this, because it makes it difficult to draw a line between documenting Python and Python modules you install.

**Comment by nuno-agostinho** [comment]
> However, isn't there a `pip`/`pip3` command too? Which one is the preferred mechanism for calling `pip`? I'm hesitant to accept this, because it makes it difficult to draw a line between documenting Python and Python modules you install.

I understand your worries, but it is the recommended method found in the official Python documentation: https://docs.python.org/3/installing

File: pages/common/python.md [hunk 1]
```diff
@@ -23,6 +23,10 @@
 
 `python -m {{module}} {{arguments}}`
 
+- Install a package:
```
Comment (sbrl) [inline pages/common/python.md:26]: I see. In that case, I guess it's ok.

Let's make this a little more explicit though, just to be on the safe side:
```suggestion
- Install a package using pip:
```

I'm not particularly experienced with Python, so a 2nd opinion from someone who is would be appreciated.

Comment (nuno-agostinho) [reply pages/common/python.md]: That seems all right, committed! :)

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @nuno-agostinho!

**Pull request merged**